Makefile

```
@@ -59,31 +87,50 @@ define SPRING_PROXY
-	./mvnw clean
+	$(MVN) clean
 
 # E.g. make install PL=swatch-core
 install: clean
-	./mvnw install -DskipTests $(if $(PL),-pl $(PL) -am)
+	$(MVN) install -DskipTests $(if $(PL),-pl $(PL) -am)
 
 # E.g. make test PL=swatch-tally TEST=TallyRetentionControllerTest,TallySnapshotControllerTest
+# The default CSS misaligns the cell values and there's no good way to get the report-only
+# task to update the CSS itself
 test:
```

checkmake (0.3.2)

[warning] Target body for "test" exceeds allowed length of 5 lines (7).

(maxbodylength)